bears/vcs/git/GitCommitBear.py

```diff
@@ -197,6 +207,36 @@ def check_body(self, body,
                                'HEAD commit. Please add one.')
             return
 
+        if host_site:
+            body_string = '\n'.join(body).lower()
+            keys = set()
+
+            if 'github' in host_site.lower():
+                keys = self.GITHUB_KEYS
+            elif 'gitlab' in host_site.lower():
+                keys = self.GITLAB_KEYS
+
+            if any(key in body_string for key in keys):
+                urls = re.findall(r'(https?://\S+)', body_string)
```

Comment:
what if the commit structure is like this
```
<tag>: shortlog

as we can see in the url: <https://......>

Fixes #109
```
your bear would search for the url in the body which is not related to `Fixes` and would yield a Result right? 
just check it after the fixes?